lib/SILGen/SILGenFunction.h

```diff
@@ -1460,6 +1460,7 @@ class LLVM_LIBRARY_VISIBILITY SILGenFunction
   ManagedValue emitTransformedValue(SILLocation loc, ManagedValue input,
                                     CanType inputType,
                                     CanType outputType,
+                                    bool isFlattening = false,
```

Comment:
We already have a way to emit a SILDeclRef with the right uncurry level -- I can dig into it if you want, let me know